cmake/module/AddLibeventIfNeeded.cmake

```diff
@@ -46,12 +46,11 @@ function(add_libevent_if_needed)
     REQUIRED IMPORTED_TARGET GLOBAL
     libevent>=${libevent_minimum_version}
   )
-  if(MINGW)
-    target_link_libraries(PkgConfig::libevent INTERFACE
-      iphlpapi
-      ws2_32
-    )
-  endif()
+  target_link_libraries(PkgConfig::libevent INTERFACE
+    # Due to a bug in the libevent build system, the required
```

Comment:
Is this tracked upstream somewhere, have we submitted a fix?

Reply:
I've added a descriptive comment.

Reply:
Shouldn't this be getting patched into depends?